lib/internal/inspector_async_hook.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const { createHook } = require('async_hooks');
+const inspector = process.binding('inspector');
+
+if (!inspector || !inspector.asyncTaskScheduled) {
+  exports.setup = function() {};
+  return;
+}
+
+const hook = createHook({
+  init(asyncId, type, triggerId, resource) {
```

Comment:
I am afraid I am not able to judge how this fact affects V8 Inspector `asyncTask*` APIs. My knowledge of these two parts (Node.js async hooks, V8 asyncTask*) is pretty limited. I am just glueing them together, so to speak.